docs/content/odh/trino/add_more_buckets.md

```diff
@@ -165,7 +165,7 @@ hive.s3.aws-secret-key=${ENV:<catalog_name_upercase>_AWS_SECRET_ACCESS_KEY}
 
 Replace all `<*>` values same as above.
 
-Add this file to `apps/kfdefs/overlays/$ENV/$CLUSTER/$TRINO_FOLDER/configs/kustomization.yaml` under the
+Add this file to `apps/kfdefs/overlays/$ENV/$CLUSTER/trino/configs/kustomization.yaml` under the
```

Comment:
make sure to get all 5 paths in this file, and use `opf-trino` instead of `trino`